frontend/src/components/indices/IndicesValidate.js

```diff
@@ -80,7 +89,7 @@ const IndicesValidate = ({ token, indicesTotalCount, isFetching, list, validate
     setIndexCount(prevIndexCount => prevIndexCount + numIndicesInSet)
 
     // load options lazily
-    list({ ...query }).then(response => {
+    listCallback({ ...query }).then(response => {
```

Comment:
I'm not sure why `listCallback` is a callback. It's just being used as a function here, but `loadData` should probably be a callback.